docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
+   - Type the following command: **`java -jar agentassist.jar`** and press **Enter**.
+   - A window similar to the below image should appear in a few seconds. You will see a graphical user interface with sample contact information already added.<br><br>
+   ![Ui](images/Ui.png)
 
-## Features Overview {#features-overview}
+4. 🎉 **Congratulations! AgentAssist is now up and running!** 🎉  
+   You're all set to start using AgentAssist to manage your contacts, track your sales, and boost your productivity! 
 
-### Feature 1: Ability to Save Current Data {#save-current-data}
+## 3.2 Graphical User Interface (GUI) Layout
 
-**Purpose:**  
-This feature ensures that any details you add to the app are saved automatically. When you close and reopen the app, all your data will still be there.
 
-**How it Works:**
-- **Command Format and Example:** Not applicable, as this process is automatic.
+To learn more about how to use commands in AgentAssist, proceed to the next section.
 
-#### Parameters
-- **Flags and Parameters:** There are no parameters needed for this feature.
+--------------------------------------------------------------------------------------------------------------------
 
-#### What to Expect
-- **If Successful:** You can access all the data you've entered previously.
-- **If There is an Error:** There's a chance that the data might not be saved due to an error, and you could lose information.
+# 4. Understanding Commands in AgentAssist {#using-agentassist}
 
----
+The true power of **AgentAssist** lies in efficiently using commands. Before diving into specific commands, let’s break down the basic structure of a command.
+
+## 4.1 Command Structure Overview
+Each command in AgentAssist consists of three key components: the **command**, **flag(s)**, and **argument(s)**.
+
+Let's take a look at the structure in more detail:
 
-### Feature 2: Add New Customer {#add-new-customer}
+| **Components**  | **Description**                                                                                                                                               | **Example**                    |
+|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------|
+| **Command**     | The action you want AgentAssist to perform.                                                                                                                   | `add`                          |
+| **Flag(s)**     | Modifiers that specify what kind of data is being handled. <br/><br/>Flag(s) are typically 1-2 letters followed by a backslash.                               | `n/`, `p/`, `r/`, `rn/`        |
+| **Argument(s)** | The values or inputs the command uses, such as client data or specific details. <br><br> The user guide may represent it as a placeholder using `<ARGUMENT>`. | `John Doe`, `john@example.com` |
```

Comment:
Perhaps replace `The user guide` with `This guide`.

Reply:
Included this change for clarity.